### PR TITLE
Test streams with dev-utils

### LINKED ISSUE #194: Improve v0.5 streaming getting started
Users trying out streaming with v0.5 should be able to install, create a streaming function, deploy it as a streaming processor, and observe the results of processing streams.

The last step is currently missing in https://projectriff.io/docs/v0.5/runtimes/streaming

One solution would be to deploy [dev-utils](https://github.com/projectriff/dev-utils) to "watch" the input and output streams.

### PR DESCRIPTION
**Pull request by jldec**
Improves getting started with the streaming runtime by adding a few additional steps to publish to a function subscribed to an input stream, and observe the output the stream

Fixes #194

[preview](https://deploy-preview-197--projectriff.netlify.com/docs/v0.5/runtimes/streaming#kafkagateway)

### CODE REVIEW

File: docs/v0.5/runtimes/streaming.md [hunk 1]
```diff
@@ -58,18 +45,16 @@ kubectl create namespace kafka
...
-#### Create Kafka gateway with riff CLI
-
 The easiest way to create a KafkaGateway is using the riff CLI:
 
 ```sh
 riff streaming kafka-gateway create franz --bootstrap-servers kafka.kafka:9092
 ```
 
-You should see a deployment and service appear:
+You should see a deployment and service:
```
Comment (scothis) [inline docs/v0.5/runtimes/streaming.md]: or add `--tail` to the create command and know it came up correctly. While this currently holds true, it's not required for the test below to pass.

File: docs/v0.5/runtimes/streaming.md [hunk 2]
```diff
@@ -116,3 +103,49 @@ riff streaming processor create time-averager \
   --input in \
   --output out
 ```
+
+## Testing with dev-utils
+
+The [dev-utils](https://github.com/projectriff/dev-utils/) container offers CLI utilities which can be used to publish messages to an input stream and subscribe to messages on an output stream.
+
+To run dev-utils in a pod in the `default` namespace, using a service account called `dev-utils`:
```
Comment (scothis) [inline docs/v0.5/runtimes/streaming.md]: please use the install instructions recommended in the dev-utils readme

File: docs/v0.5/runtimes/streaming.md [hunk 3]
```diff
@@ -116,3 +103,49 @@ riff streaming processor create time-averager \
...
+```bash
+kubectl create serviceaccount dev-utils
+kubectl create rolebinding dev-utils-edit --clusterrole=edit --serviceaccount=default:dev-utils
+kubectl run dev-utils --serviceaccount=dev-utils --generator=run-pod/v1 --image=projectriff/dev-utils
+```
+
+Using separate terminal windows, subscribe to the `in` and `out` streams.
+
+```bash
+kubectl exec dev-utils -it -- subscribe in --payload-encoding raw
```
Comment (scothis) [inline docs/v0.5/runtimes/streaming.md]: if we have people subscribe to `in` we should do something with it

File: docs/v0.5/runtimes/streaming.md (hunk 2 above)
Comment (jldec) [reply docs/v0.5/runtimes/streaming.md]: @scothis those instructions introduce yet another name for users to understand.
I'd prefer to use the name 'dev-utils' for the pod and the service account.
Happy to submit a PR for the dev-utils README.

File: docs/v0.5/runtimes/streaming.md (hunk 3 above)
Comment (jldec) [reply docs/v0.5/runtimes/streaming.md]: We publish to it - I think it's reassuring to see the things that you publish from the CLI appear somewhere.

File: docs/v0.5/runtimes/streaming.md (hunk 2 above)
Comment (trisberg) [reply docs/v0.5/runtimes/streaming.md]: using a riff- prefix identifies the resource better, dev-utils is very generic

Comment (scothis) [reply docs/v0.5/runtimes/streaming.md]: as @trisberg points out, the name `dev-utils` is very generic and it won't be obvious looking at the pod what it is. The name change in dev-utils repo was intentional.

File: docs/v0.5/runtimes/streaming.md (hunk 3 above)
Comment (scothis) [reply docs/v0.5/runtimes/streaming.md]: Then a quick note saying that the user should expect to see the messages they publish in this console would help. Right now there's no context for what the command does or what is expected.

File: docs/v0.5/runtimes/streaming.md (hunk 2 above)
Comment (jldec) [reply docs/v0.5/runtimes/streaming.md]: ok

**Pull request merged**